.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
-.DS_Store
\ No newline at end of file
+.DS_Store
+.git
```

Comment:
You want to ignore the git folder itself?

Reply:
Let  me  revert this. Was futzing  with stuff toget t hings  to work